.cache/roles/google_cloud_platform.google_cloud_ops_agents

```diff
@@ -0,0 +1 @@
+../..
```

Comment:
is this included on purpose? It's very obscure and ideally would have a comment.